wolfssl/wolfcrypt/settings.h

```diff
@@ -188,6 +188,11 @@
 
 #ifdef WOLFSSL_USER_SETTINGS
     #include "user_settings.h"
+#else
+    /* include the options.h, which is generated from ./configure */
+    #if defined(HAVE_CONFIG_H) && !defined(NO_WOLF_OPTIONS_H)
+        #include <wolfssl/options.h>
```

Comment:
options.h is for user apps to have the same settings as the wolfssl library.  Why would we include it in the library proper?